### ISSUE TITLE
Названия событий в socket.io выглядят странно

### ISSUE THREAD

**Issue by andrewQwer** [Joiler/SimpleChat#8]
Обычно именуются через разделители типа:
`socket.on('user:disconnected',...)`
`socket.on('message:sent',...)`
`socket.on('message:received',...)`

**Comment by Joiler** [comment]
Согласен, поменял. Как ни странно, именно такие названия типа 'read message', 'chat message' встречались в Интернете и документации.

**Comment by andrewQwer** [comment]
> Как ни странно, именно такие названия типа 'read message', 'chat message' встречались в Интернете и документации.

может для простоты, не знаю
